content/purgatorio/launch-and-mantain-the-system/automation-and-reproducibility.md

```diff
@@ -1,5 +1,242 @@
 ---
 title: Automation and Reproducibility
 author: clone95
-description: ---
----
\ No newline at end of file
+description: This guide introduces you to the best practices of the Data Science process automation and research reproducibility.
+---
+
+# Index 
+
+- [Advantages of Automation and The Need for Reproducibility](#Advantages-of-Automation-and-The-Need-for-Reproducibility)
+- [Automation and Reproducibility for the Data Science Process](#Automation-and-Reproducibility-for-the-Data-Science-Process)
+- [AutoML](#AutoML)
+- [No Free Lunch and Black Boxes](#No-Free-Lunch-and-Black-Boxes)
+
+## Advantages of Automation and The Need for Reproducibility
+
+All the previous guides within Purgatorio suggest various things:
+
+- there are _many steps_ to be taken to start from the raw data and get to high-performance models 
+- you can make a lot of _mistakes_ and various kinds of mistakes.
+- many steps can be _tedious or boring_ to repeat manually (Data Cleaning, I'm watching you).
+
+So, how can we simplify our work and become more productive, decrease the likelihood of making mistakes, and make our work repeatable? 
+
+The keyword is one: [Automation](https://en.wikipedia.org/wiki/Automation). 
+
+Automation is the best solution to all the problems mentioned above, and also saves us time! 
+
+This is true for almost every field (not only Data Science), in fact, if we look at the Wikipedia definition it says: 
+
+ **_Advantages of Automation_**:
+
+- Increased throughput or productivity.
+- Improved quality or increased predictability of quality.
+- Improved robustness (consistency), of processes or products.
+- Increased consistency of output.
+- Reduced direct human labor costs and expenses.
+- Replaces human operators in tasks that involve hard physical or monotonous work (e.g., using 
+- Performs tasks that are beyond human capabilities of size, weight, speed, endurance, etc.
+- Reduces operation time and work handling time significantly.
+- Frees up workers to take on other roles.
+- It provides higher-level jobs in the development, deployment, maintenance, and running of the automated processes.
+
+This article explains very well why Automation is crucial for the Data Science world:
+
+- [4 Ways Automation Is Altering Data Science](https://towardsdatascience.com/4-ways-automation-is-altering-data-science-36a0d2755e4f)
+
+### Disadvantages of Automation
+
+Some might argue that Automation has not only advantages but also disadvantages, such as job losses (it takes less human to do the same things).
```

Comment:
So the counter-argument of job replacements or perceived job loss is that those replaced employees can be upskilled or used for other tasks, and redundant tasks can be replaced in this manner.

Historically, automation has only created more jobs because it opens up more possibilities and more value-added services get created due to the fact, which need manual labour/human intervention. So job loss isn't fully true - try to put a succint version of this as counter-argument cause it's important for the readers to know.

Reply:
this is well described in the links linked. exactly!